src/Command.js

```diff
@@ -19,6 +19,47 @@ export default class Command {
     this.concurrency = (!flags || flags.concurrency === undefined) ? DEFAULT_CONCURRENCY : Math.max(1, +flags.concurrency || DEFAULT_CONCURRENCY);
   }
 
+  get name() {
+    // For a class named "FooCommand" this returns "foo".
+    return this.className.replace("Command", "").toLowerCase();
+  }
+
+  get className() {
+    return this.constructor.name;
```

Comment:
and in the future, when `asini` is an `asini` repo, custom commands should be possible, and names could _potentially_ clash. it might be nice to be explicit with the name rather than relying on the class name.